shared/charts/couchdb-prometheus-exporter/README.md

```diff
@@ -0,0 +1,58 @@
+# CouchDB Prometheus Exporter
+
+[couchdb-prometheus-exporter](https://github.com/gesellix/couchdb-prometheus-exporter) is a [CouchDB](http://couchdb.apache.org/) metrics exporter for [Prometheus](http://prometheus.io/)
+
+The CouchDB metrics exporter requests the CouchDB stats from the `/_stats` and `/_active_tasks` endpoints and exposes them for Prometheus consumption. You can optionally monitor detailed database stats like disk and data size to monitor the storage overhead. The exporter can be configured via program parameters, environment variables, and config file.
+
+
+## TL;DR;
+
+```console
+$ helm install path_to_chart/couchdb-prometheus-exporter
+```
+
+## Introduction
+
+This chart bootstraps a CouchDB Prometheus Exporter deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+  - Kubernetes 1.8+ with Beta APIs enabled
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release path_to_chart/couchdb-prometheus-exporter
+```
+
+The command deploys couchdb-prometheus-exporter on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the gpii-preferences chart and their default values.
+
+Parameter | Description | Default
+--- | --- | ---
+`replicaCount` | desired number of pods | `1`
```

Comment:
hm, why do we use `camelCase` here  vs `under_score` naming convention for other variables?

Reply:
I'll look at this in the follow-up cleanup PR. Looks like camel case is the convention for Charts.